README.md

```diff
@@ -392,22 +394,18 @@ A basic usage of this object template would involve the creation of `templates/u
 
 ```
 
-Please note that the UserAssignedIdentity name is set internally. This value is set in the `adp-flux-services` repository which follows standard naming conventions "{TEAM_MI_PREFIX}-{SERVICE_NAME}". For e.g. In SND1 if the `TEAM_MI_PREFIX` value is set to 'sndadpinfmid1401' and `SERVICE_NAME` value is set to 'adp-demo-service', then `managedIdName` value will be : sndadpinfmid1401-adp-demo-service .
+This template does not require you to supply any required values in the values.yaml in the file.
+
+This template uses the below values, whose values are set using platform variables in the `adp-flux-services` repository as a part of the service's ASO helmrelease value configuration, and you don't need to set them explicitly in the values.yaml file.
 
-This template uses the below platform variables internally, and you don't need to set them explicitly in the values.yaml file. It must be setup in the `adp-flux-services` repository as a part of the service's ASO helmrelease value configuration.
-- TEAM_MI_PREFIX
-- SERVICE_NAME
+- teamMIPrefix
+- serviceName
 - teamResourceGroupName
 - clusterOIDCIssuerUrl
 
-#### Required values
+UserAssignedIdentity Name is derived internally, and it is set to = `{TEAM_MI_PREFIX}-{SERVICE_NAME}`
 
-The following values need to be set in the parent chart's `values.yaml` in addition to the globally required values [listed above](#all-template-required-values).
-
-```
-userAssignedIdentity: {}
-
-```
+For e.g. In SND1 if the `TEAM_MI_PREFIX` value is set to "sndadpinfmid1401" and `SERVICE_NAME` value is set to "adp-demo-service", then `UserAssignedIdentity` value will be : "sndadpinfmid1401-adp-demo-service".
```

Comment:
can users supply create a user MI with an optional ending suffix?>

Reply:
in any way? i.e. one requirement is to mandate convention as above, but also need to be able to create multiple MIs (tier 3)s, provided they can do that with a name at the end?

Reply:
a team created MI of sndadpinfmid1401-adp-demo-service-**queuereader**

Reply:
@DaRoszko : In current implementation we are just allowing to setup one Identity per service which will have all the assignments like 'queuereader', 'topicsender', 'keyvaultsecretuser'

I dont see a need that service would want to setup more that one identity?

for e.g. In FFC as well there is one identity for each service.
![image](https://github.com/DEFRA/adp-aso-helm-library/assets/89150034/5e95d027-04f9-4a33-bb4c-756b939c48a6)

Reply:
i don't think this is correct: have you checked current ffc live implementation? I believe a service can create one or more MIs for their service.

Reply:
Can we validate current practice. And need to map to use case: A service could want to generate multiple MIs

Reply:
@kbabigumira  @ajaysoni-cap  views?

Reply:
original views like KV role would be their default MI, and a default MI for app config. look at this 
![image](https://github.com/DEFRA/adp-aso-helm-library/assets/6905189/a11dd5e8-80a5-4606-8d33-7ff11d0f8d0d)

Reply:
@DaRoszko :
I can see for each MI in above list has got github repository with that name which i think represent one service for that team. 

So if I see 'pay' service from ADP side, my understanding is - we will need to setup folder for each of them in ADP-FLUX-SERVICES repository and which will create all the above identities

Reply:
Interesting. So every single one across FFC is 1-1 mapping

Reply:
yeah

Reply:
yes, 1 repo, 1 service to answer earlier question

Reply:
Resolving as per discussion